sql/core/src/main/java/org/apache/spark/sql/sources/v2/reader/Scan.java

```diff
@@ -90,8 +88,8 @@ default MicroBatchStream toMicroBatchStream(String checkpointLocation) {
   /**
    * Returns the physical representation of this scan for streaming query with continuous mode. By
    * default this method throws exception, data sources must overwrite this method to provide an
-   * implementation, if the {@link Table} that creates this scan implements
-   * {@link SupportsContinuousRead}.
+   * implementation, if the {@link Table} that creates this scan returns
+   * {@link TableCapability#CONTINUOUS_READ} support in its {@link Table#capabilities()}.
```

Comment:
These are non-functional changes and should not be in this commit.

Reply:
How can we move this? `SupportsContinuousRead` is removed in this PR and `TableCapability#CONTINUOUS_READ` is added in this PR.

Reply:
You're right, sorry I missed the links that are removed. This change is fine.